modules/ui.py

```diff
@@ -566,19 +566,6 @@ def apply_setting(key, value):
     return value
 
 
-def update_generation_info(args):
-    generation_info, html_info, img_index = args
-    try:
-        generation_info = json.loads(generation_info)
-        if img_index < 0 or img_index >= len(generation_info["infotexts"]):
-            return html_info
-        return plaintext_to_html(generation_info["infotexts"][img_index])
-    except Exception:
-        pass
-    # if the json parse or anything else fails, just return the old html_info
-    return html_info
-
-

```

Comment:
Just a heads-up, it looks like you may be undoing unrelated changes made in PR 4573?